UNRELEASED.md

```diff
@@ -7,6 +7,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 ### Enhancements
 
 - Update build toolchain to use Babel v7, PostCSS v7 and Rollup v1. Update our build targets match our [supported browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers), leading to a reduction in bundle size ([#837](https://github.com/Shopify/polaris-react/pull/837))
+- Updated `Toast` to use App Bridge `Toast` action and remove deprecated `isDismissible` property
```

Comment:
```suggestion
```

We can drop this as it is strictly an implementation detail. I think the dep upgrade below gives us enough context on the change.